### ISSUE TITLE
Karabiner Elements not working

### ISSUE THREAD

**Issue by alvst** [pqrs-org/Karabiner-Elements#3204]
I haven't been able to get Karabiner to work the last couple days. It started when I updated to the latest version of the app but even after reverting to previous versions, I am still having he same problem. I'm not sure if this started after a [macOS update](https://karabiner-elements.pqrs.org/docs/help/troubleshooting/stopped-working-after-macos-update/) but regardless even after doing all the steps described multiple times, I still am not able to get Karabiner/my keyboard to work when Karabiner is running. My Keyboard inputs show up in EventViewer don't effect anything else in the system (briefly, before Karabiner starts, it does work). Any way I can fix this? (Again I've restarted and reinstalled Karabiner.) I am running macOS Monterey 12.5.1.

**Comment by frankcbliu** [comment]
+1, the same problem. my mac version also is macOS Monterey 12.5.1

**Comment by frankcbliu** [comment]
> +1, the same problem. my mac version also is macOS Monterey 12.5.1

when i restart my mac, i found that the problem disappear.

**Comment by toxaO** [comment]
+1, the same problem.
my mac version also is macOS Monterey 12.5.1
I restart my mac, but the problem doesnt disappear.

**Comment by alvst** [comment]
Seems others are having this problem but I was able to solve this for my be restarting in safe mode and opening up app by app that affects Bluetooth/External Devices. This got me to realize that the problem was being caused by a conflict between [USB Overdrive](https://www.usboverdrive.com) and Karabiner where USB Overdrive was overwriting my keys and causing my keys to no longer be recognized. I uninstalled USB Overdrive and instead moved to using Keyboard Maestro for the functions (remapping a couple keys not recognized by Karabiner) and that's fixed my problem.  This problem seems to have been caused by a recent USB Overdrive update or macOS update, not sure which.
